webapp/src/pages/home/Home.js

```diff
@@ -81,49 +80,42 @@ class Home extends Component {
   }
 
   render() {
-    let table = (<div></div>)
-
-    if (this.state.rows && this.state.rows.length > 0) {
-      const theadMarkup = (
-        <tr>
-          {this.state.headings.map((_cell, cellIndex) => {
-            return (
-              <th className="Cell">
-                {this.state.headings[cellIndex]}
-              </th>
-            )
-          })}
-        </tr>
-      );
-
-      const tbodyMarkup = this.state.rows.map((_row, rowIndex) => {
-        return (
-          <tr>
-            {fieldNames.map((_cell, cellIndex) => {
-              return (
-                <td className="Cell">
-                  {
-                    this.state.rows[rowIndex][fieldNames[cellIndex]] === "Apply now" || 
-                      this.state.rows[rowIndex][fieldNames[cellIndex]] === "Continue application" ?
-                      <NavLink to="/applicationForm">{this.state.rows[rowIndex][fieldNames[cellIndex]]}</NavLink> :
-                      this.state.rows[rowIndex][fieldNames[cellIndex]]
-                  }
-                </td>
-              )
-            })}
-          </tr>
-        )
-      });
-
-      table = this.state.rows ? (
-        <table align="center" className="Table">
-          <thead>{theadMarkup}</thead>
-          <tbody>{tbodyMarkup}</tbody>
-        </table>
-      ) : (<div></div>)
-    }
 
-    let statusClass = this.state.applicationStatus == "Submitted" 
+    // if (this.state.rows && this.state.rows.length > 0) {
```

Comment:
Hey man. Why was this commented out?

Reply:
It was never used => decided to comment it out because maybe someone is working on it and plan to finish it out later. I comment on it with 'unused'

Reply:
Yeah, the setState on rows was commented out a while ago, so the table didn't render.